### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by ealmonte32**
system suggested to remove the executable bit, 0644 should work fine without throwing alert about marked for execution

### CODE REVIEW

File: Dockerfile.template [hunk 1]
```diff
@@ -9,6 +9,7 @@ RUN apt-get update && \
         libffi-dev \
         libssl-dev \
         matchbox \
+        nano \
```
Comment (vpetersson) [inline Dockerfile.template]: Nooooo :) 
Never put an editor in a Dockerfile.

File: Dockerfile.template [hunk 2]
```diff
@@ -19,10 +20,12 @@ RUN apt-get update && \
         python-simplejson \
         libraspberrypi0 \
         ifupdown \
+        python-urllib3 \
         sqlite3 \
         uzbl \
         x11-xserver-utils \
         xserver-xorg && \
+    apt full-upgrade \
```
Comment (vpetersson) [inline Dockerfile.template]: This goes against Docker best-practices, which states that you rely on the upstream image for this.

**Comment by vpetersson** [comment]
Tests are failing, but also, why is `install.sh` in there?

File: Dockerfile.template (hunk 1 above)
Comment (ealmonte32) [reply Dockerfile.template]: lol how did this even get here.. these were some of the tests dockerfile lines i was using messing with the container itself .. ugh i must of pressed the wrong thing on my forked repo.. sorry now I have to learn how to only edit on directly this branch

**Comment by ealmonte32** [comment]
omg cancel this lol

**Comment by ealmonte32** [comment]
i was messing with my forked version and i must have pressed the wrong button and it sent all these changes over.. :(

**Comment by ealmonte32** [comment]
I literally just wanted to change two lines in the ansible / main.yml file, how did all those commits end up part of that PR as well.. ughhhhhh

**Comment by ealmonte32** [comment]
I was trying to delete some patch-# branches that keep getting created every time I edit any of the master branch files here, but I wanted to delete all those patches after merging them with my fork, like in this screenshot:
image

So I dont know what I pressed for them to try and become commits on this branch..

**Pull request closed without merging**